layouts/shortcodes/feature-state.html

```diff
@@ -33,7 +33,7 @@
 
             <div class="feature-state-notice feature-{{ .stage }}" title="{{ printf "%s %s" (T "feature_state_feature_gate_tooltip") $feature_gate_name }}">
               <span class="feature-state-name">{{ T "feature_state" }}</span> 
-              <code>{{ T "feature_state_kubernetes_label" }} v{{ .fromVersion }} [{{ .stage }}]</code>
+              <code>{{ T "feature_state_kubernetes_label" }} v{{ .fromVersion }} [{{ .stage }}] Enabled: {{.defaultValue}}</code>
```

Comment:
We need to localize strings here.
I suggest two localized strings. The English versions could be:
`Enabled by default in Kubernetes {{ .version }}`
`Disabled by default in Kubernetes {{ .version }}`

and we could use a `<details>` element to allow revealing this plus other information. WDYT @vinayakankugoyal ?

See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details about the  `<details>` element.

---

Regardless, we _must_ support localizing the strings we use here.

Reply:
I would much prefer just showing the information inline, what other information is there to display?

How I would go about localizing this?

Reply:
> How I would go about localizing this?

@vinayakankugoyal To localize the hardcoded string "Enabled" _(or any other string)_, simply add it to the **[en.toml](https://github.com/kubernetes/website/blob/main/data/i18n/en/en.toml)** file. Then, utilize the Hugo translate  'T' [function](https://gohugo.io/functions/lang/translate/) to fetch its value.  

Check out how [`feature_state_kubernetes_label`](https://github.com/kubernetes/website/blob/2f018d48603466c2a9a4a7891dda8cb581669128/data/i18n/en/en.toml#L200) has been used in the same file for guidance.

Reply:
Updated. Thanks for the pointers!